src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -282,6 +282,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageReference Include="NuGet.Packaging" Version="5.11.6" />
     <PackageReference Include="System.Formats.Asn1" Version="6.0.1" />
+    <PackageReference Include="System.Text.Json" Version="8.0.10" />
```

Comment:
The nuget does not have version 8.0.10 - https://www.nuget.org/packages/System.Text.Json but on the CVE link they mentioned to upgrade to 8.0.10 - https://github.com/advisories/GHSA-8g4q-xg66-9fp4

Reply:
Hmm but the build succeeds still so I think it might be in one of the internal feeds that the core tools project uses today